### ISSUE TITLE
OutboundProfile is referencing a unknown AuthenticationProfile: '_default'

### ISSUE THREAD

**Issue by edward1f** [Axway-API-Management-Plus/apim-cli#232]
**API-Mgr. CLI version**
> 1.4.0

**API-Manager and Service-Pack Version**
7.7.20210530

**Expected behavior**
Success for import the configuration

**Actual behavior**
240    ERROR         APIImportApp| Cannot validate/fulfill configuration file.
                                 | OutboundProfile is referencing a unknown AuthenticationProfile: '_default'

Attached the api-config file which generates the error
[api-config.txt](https://github.com/Axway-API-Management-Plus/apim-cli/files/7465336/api-config.txt)

**Comment by cwiechmann** [comment]
Hi @edward1f, This is a known issue: #220 and it has been fixed with version 1.4.0. However, I tried to reproduce the issue anyway, but I was able to import the API successfully:
```
----------------------------------------------------------------------------------------
API-Manager CLI: 1.4.0

To report issues or get help, please visit:
https://github.com/Axway-API-Management-Plus/apim-cli
----------------------------------------------------------------------------------------
Module: API - I M P O R T (1.4.0)
----------------------------------------------------------------------------------------
0      INFO     APIManagerAdapter| Successfully connected to API-Manager (7.7.20210830) on: https://xxxxxxxxxxxx:443
326    INFO  IImportConfigAdapter| Reading API-Definition (Swagger/WSDL) from file: 'C:\Axway\Tools\apim-cli-1.4.0\samples\petstore.json' (absolute path)
363    INFO      APISpecification| Used the backendBasepath: 'http://localhost:8080/invoice/v1' to adjust the API-Specification.
547    INFO      APIImportManager| No existing API found, creating new!
1127   INFO          CreateNewAPI| Create published API: 'E-Invoice' 2.0 based on Swagger 2.0 specification.
2679   INFO          CreateNewAPI| Successfully created published API: 'E-Invoice' 2.0 (ID: e49eed2d-aecf-433f-a181-2283e490af47)
```

Can you please double check, you tried with version 1.4.0?

**Comment by edward1f** [comment]
Hi @cwiechmann ,

Sorry, this is my mistakes. Currently, I tried to use the CLI Java-APIs and deploy via jenkins. I forgot to change apimcli-apis version to 1.4.0 in my file pom.xml. Currently, it worked. You can close this issue. 
Thank you.
